odbc/tests/connection_tests.rs

```diff
@@ -121,31 +121,4 @@ mod integration {
         expected_error_message =
             "[MongoDB][API] Missing property \"Driver\" or \"DSN\" in connection string"
     );
-    test_connection_diagnostics!(
```

Comment:
Can we keep these tests and update the expected SQL return to be SQL_SUCESS, for completeness.